mbed_lstools/platform_database.py

```diff
@@ -251,6 +253,40 @@
     u'RIOT': u'RIOT',
 }
 
+
+def _get_modified_time(path):
+    try:
+        mtime = getmtime(path)
+    except OSError:
+        mtime = 0
+    return datetime.date.fromtimestamp(mtime)
+
+
+def _older_than_me(path):
+    return _get_modified_time(path) < _get_modified_time(__file__)
+
+
+def _overwrite_or_open(db):
+    try:
+        if db is LOCAL_PLATFORM_DATABASE and _older_than_me(db):
+            raise ValueError("Platform Database is out of date")
+        with open(db, encoding="utf-8") as db_in:
+            return json.load(db_in)
+    except (IOError, ValueError) as exc:
+        if db is LOCAL_PLATFORM_DATABASE:
+            logger.warning(
+                "Error loading database %s: %s; Recereating", db, str(exc))
```

Comment:
Lil typo here: Recereating ->Recreating

Reply:
Thanks!

Reply:
The branch name also contains a typo, FYI.